app/views/portal/index.rb

```diff
@@ -4,6 +4,7 @@ module Portal
   class Index < ApplicationView
     include SearchableView
     include HeroImageDisplayingView
+    include PaginatedView
```

Comment:
I have added a few specs for this, split over shared examples and unit-level tests of the concern itself, which approach we could use as the basis for a general strategy for testing views.